utils/eventtracking.js

```diff
@@ -82,10 +85,18 @@ const track = async (eventType, eventProperties, options) => {
     }
 };
 
+async function getUserCountry() {
+    return fetch('https://ipinfo.io/json').then(
+        (response) => response.json()
+    ).then(
+        (jsonResponse) => jsonResponse.country
+    ).catch(error => console.log(error));
```

Comment:
Any difference if we use `console.error` over `console.log` here?

Reply:
Addressed above.